src/app/Main/Body/QueryPage/Services/ContainerServices/generateDataObjects.services.tsx

```diff
@@ -0,0 +1,87 @@
+import { TFunction } from "i18next";
+import {
+    ClientIsolateCountedGroups,
+} from "../../../../../Shared/Model/Client_Isolate.model";
+import { adaptCountedIsolatesGroupsService } from "../adaptCountedIsolatesGroups.service";
+import { calculateRelativeTableData } from "../TableServices/calculateRelativeTableData.service";
+import {
+    calculateRowColSumsAbsolute,
+    calculateRowColSumsRelative,
+} from "../TableServices/calculateRowColSums.service";
+import { generateStatisticTableDataAbsService } from "../TableServices/generateStatisticTableDataAbs.service";
+import { generateTableHeaderValuesService } from "../TableServices/generateTableHeaderValues.service";
+import { FilterInterface } from "../../../../../Shared/Model/Filter.model";
+
+export function generateDataObjectsService(props: {

```

Comment:
Name could be more explicit. E.g. generateStatisticData or so.

Reply:
I think the name is okay. Because it's not only the statistic data that is generated here.

Reply:
I think not that the name is ok. It could be named generateSomething without loosing any information what this service does.

Reply:
But it was not reject, so you can leave it as it is